modules/flowable-dmn-spring-configurator/src/main/java/org/flowable/dmn/spring/configurator/SpringDmnEngineConfigurator.java

```diff
@@ -19,7 +19,6 @@
 import org.flowable.dmn.engine.DmnEngine;
```

Comment:
Just above this at line 17 the import, `org.flowable.common.engine.impl.el.ExpressionManager;` is no longer used.

Reply:
Thanks for the check David